src/test/unit/profile-data.test.js

```diff
@@ -893,24 +893,24 @@ describe('getTimingsForPath in a non-inverted tree', function() {
         selfTime: {
           value: 2,
           breakdownByImplementation: { ion: 1, baseline: 1 },
-          breakdownByCategory: withSingleSubcategory([0, 2, 0, 0, 0, 0, 0, 0]), // [Idle, Other, ...]
+          breakdownByCategory: withSingleSubcategory([0, 0, 0, 2, 0, 0, 0, 0]), // [Idle, Other, Layout, JavaScript, ...]
```

Comment:
Not for this PR, general comment: These numbers felt very magical coming into the review. It seems like it would be better to use named variables instead.

Reply:
In the next PR, prettier puts them on several lines which makes it possible to add a comment in front of each, which is probably better indeed.